Compat408.ml

```diff
@@ -0,0 +1,11 @@
+(************************************************************************)
+(*   FlexDLL                                                            *)
+(*   Alain Frisch                                                       *)
+(*                                                                      *)
+(*   Copyright 2007 Institut National de Recherche en Informatique et   *)
+(*   en Automatique.                                                    *)
+(************************************************************************)
+
+module Stdlib = struct
+  let compare = Pervasives.compare
+end
```

Comment:
Can't this just be...
```suggestion
module Stdlib = Pervasives
```
... and then any future uses of `Stdlib.` should be safe(ish) too?

Reply:
I change as suggested, original I just wanted to make sure that only the function needed are imported, but you are right it might be more useful to directly import the whole module.